.github/workflows/ci.yml

```diff
@@ -13,80 +13,54 @@ jobs:
     name: run tests
     strategy:
       matrix:
-        platform: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Hm, running two versions of Ubuntu seems overkill. I've thought about it some more and think that your initial suggestion is correct and we should use `-latest` everywhere.